python/ray/rllib/agents/agent.py

```diff
@@ -177,6 +177,9 @@
         # Optional whitelist of policies to train, or None for all policies.
         "policies_to_train": None,
     },
+
+    # Whether environments are in remote process or not
+    "remote_worker_envs": False,
```

Comment:
Can you move this to under the Environment config section?

Reply:
Done.